float2words/float2words.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+
+from num2words import num2words
+from decimal import Decimal, ROUND_HALF_UP
+
+
+def float2words(number, lang, sfx1='', sfx2='', connector=', ',
+                precision=2):
+    """Convert float to words by given optional parameters.
+
+    Args:
+        number (float): float number.
+        lang (str): language code (e.g. 'en_US', 'ru_RU', etc.).
+        sfx1 (str): suffix for whole part of float number
+            (e.g. 'USD', '$', 'kg', etc.).
+        sfx2 (str): suffix for decimal part of float number.
+            (e.g. 'ct.', 'g', etc.)
+        connector (str): connector will be used between whole and
+            decimal parts expressed in words.
+        precision (int): number of digits after decimal point.
+
+    Returns:
+        number_in_words (str): float number in words.
+
+    """
+    whole_part, decimal_part = _split_float(number, precision)
+    number_in_words = ''.join([
+        _convert_and_combine(whole_part, lang, sfx1),
+        connector,
+        _convert_and_combine(decimal_part, lang, sfx2)
+    ])
+    return number_in_words
+
+
+def _split_float(number, precision):
```

Comment:
Incorrect methods order. Methods must go in order they depend on each other. `float2words` depends on `_split_float`, so `_split_float` must be defined first.